### ISSUE TITLE
m.image thumbnail_info with missing fields not rendered

### ISSUE THREAD

**Issue by auscompgeek** [vector-im/riotX-android#1209]
If the `mimetype` field is missing from `thumbnail_info`, RiotX displays "Malformed event, cannot display", despite not being malformed per the spec.

Example content:
```json
{
  "msgtype": "m.image",
  "body": "tex.png",
  "url": "mxc://vovo.id.au/mQteIHfVUBSefcmexHFjsDKj",
  "info": {
    "mimetype": "image/png",
    "size": 10638,
    "thumbnail_info": {
      "size": 2800,
      "h": 65,
      "w": 188
    },
    "thumbnail_url": "mxc://vovo.id.au/cwACVJifVCiwhDMATIYppuCP",
    "h": 189,
    "w": 561
  }
}
```

**Comment by bmarty** [comment]
Good catch. I've fixed it for the next release

**Comment by bmarty** [comment]
Fixed in #1238
